### ISSUE TITLE
Cannot destructure from `super`

### ISSUE THREAD

**Issue by sandersn** [Microsoft/TypeScript#7260]
**TypeScript Version:**  master's HEAD

**Code**

```ts
class K {
  m() { }
}
class C extends K {
  m2() {
    let { m: f } = super; // error
    let { m } = this; // ok
  }
}
```

**Expected behavior:** No error when destructuring from `super`.

**Actual behavior:** Error: 'super' must be followed by an argument list or member access.

**Comment by sandersn** [comment]
Never mind, this is a limitation imposed by the ecmascript spec. It's external.
